app/controllers/ballpark_logs_controller.rb

```diff
@@ -1,4 +1,5 @@
 class BallparkLogsController < ApplicationController
+  before_action :authenticate_user,{only:[:new,:create,:edit,:update,:delete]}
```

Comment:
```suggestion
before_action :authenticate_user, only:[:new,:create,:edit,:update,:delete]
```

before_actionのオプションをハッシュで囲うのはあまりみたことがないので`{}`外して良さそうです！